### PR TITLE
Fixed formatting, improved readability, fixed capitalization, made comments uniform

### PR DESCRIPTION
**Pull request by ScureX**
Removed unnecessary line breaks to improve readability as well as making comments more uniform.

### CODE REVIEW

File: kernel-open/nvidia/nv.c [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/*
+3/*
```
Comment (Uzlopak) [inline kernel-open/nvidia/nv.c:1]: Seems wrong
```suggestion
/*
```

Comment (ScureX) [reply kernel-open/nvidia/nv.c]: Absolutely right, must've hit my mouse button on accident, my bad x)

**Comment by mtijanic** [comment]
Thank you for the PR, but marking as `cosmetic-postponed` for now. Please see https://github.com/NVIDIA/open-gpu-kernel-modules/discussions/190 for details.

**Comment by mtijanic** [comment]
Thank you for the PR, but marking as `cosmetic-postponed` for now. Please see https://github.com/NVIDIA/open-gpu-kernel-modules/discussions/190 for details.

**Comment by niv** [comment]
Thank you for your interest in contributing to open-gpu-kernel-modules.

As recently explained in [CONTRIBUTING.md](https://github.com/NVIDIA/open-gpu-kernel-modules/blob/main/CONTRIBUTING.md), we have decided to not accept cosmetic changes for now.

All changes going into this repository incur significant overhead on our end, and we'd like to focus our resources on executable code changes; so we have decided to close all PRs that make no code changes.

If and when this position changes, your PR will still be in the archives and revisited (as long as it's still tagged cosmetic-postponed).

Please do not feel discouraged to submit other changes! In addition, we will be happy to keep the conversation going on the Discussion boards here on GitHub, and elsewhere.

**Pull request closed without merging**